### ISSUE TITLE
Update the test suite and add as workflow

### ISSUE THREAD

**Issue by WeisLeDocto** [LaboratoireMecaniqueLille/crappy#57]
A test suite was added to Crappy in a2a2043c.
It is currently still a draft, and needs to be updated in order to be used.
Only the core functionalities will be tested, so that the tests can run without any hardware.

Once the test suite is ready, it will be added as a GitHub workflow to run it automatically at each new PR.

**Comment by WeisLeDocto** [comment]
- [x] Camera configuration interface
- [x] Parent Block
- [x] Simple runnable examples (ones from the docs ?)
- [x] Main Blocks
- [x] Base Actuator + fake ones
- [x] Base InOut + fake ones
- [x] Base Camera + fake ones
- [x] Image processing tools
